assets/models/system/phi-3-mini-4k-instruct/description.md

```diff
@@ -0,0 +1,187 @@
+
+
+## Benchmarks
+
+We report the results for Phi-3-Mini-4K-Instruct on standard open-source benchmarks measuring the model's reasoning ability (both common sense reasoning and logical reasoning). We compare to Phi-2, Mistral-7b-v0.1, Mixtral-8x7b, Gemma 7B, and GPT-3.5.
+
+All the reported numbers are produced with the exact same pipeline to ensure that the numbers are comparable. These numbers might differ from other published numbers due to slightly different choices in the evaluation.
+
+As is now standard, we use few-shot prompts to evaluate the models, at temperature 0. 
+The prompts and number of shots are part of a Microsoft internal tool to evaluate language models, and in particular we did no optimization to the pipeline for Phi-3.
+More specifically, we do not change prompts, pick different few-shot examples, change prompt format, or do any other form of optimization for the model.
+
+The number of k–shot examples is listed per-benchmark. 
+
+|| Phi-3-Mini-4K-In<br>3.8b (this model) | Phi-3-Small<br>7b (preview) | Phi-3-Medium<br>14b (preview) | Phi-2<br>2.7b | Mistral<br>7b | Gemma<br>7b | Mixtral<br>8x7b | GPT-3.5<br>version 1106 |
+|---|---|---|---|---|---|---|---|---|
+| MMLU <br>5-Shot |68.80|75.60|78.20|56.3|61.70|63.60|70.50|71.40|
+| HellaSwag <br> 5-Shot |76.70|78.70|83.00|53.6|58.50|49.80|70.40|78.80|
+| ANLI <br> 7-Shot |52.80|55.00|58.70|42.5|47.10|48.70|55.20|58.10|
+| GSM-8K <br> 0-Shot; CoT |82.50|88.90|90.30|61.1|46.40|59.80|64.70|78.10|
+| MedQA <br> 2-Shot |70.00|58.20|69.40|40.9|50.00|49.60|62.20|63.40|
+| AGIEval <br> 0-Shot |37.50|45.00|48.40|29.8|35.10|42.10|45.20|48.40|
+| TriviaQA <br> 5-Shot |64.00|59.10|75.60|45.2|75.20|72.30|82.20|85.80|
+| Arc-C <br> 10-Shot |84.90|90.70|91.00|75.9|78.60|78.30|87.30|87.40|
+| Arc-E <br> 10-Shot |94.60|97.10|97.80|88.5|90.60|91.40|95.60|96.30|
+| PIQA <br> 5-Shot |83.20|87.80|87.70|60.2|77.70|78.10|86.00|86.60|
+| SociQA <br> 5-Shot |76.60|79.00|80.20|68.3|74.60|65.50|75.90|68.30|
+| BigBench-Hard <br> 0-Shot |71.70|74.90|81.30|59.4|57.30|59.60|69.70|68.30|
+| WinoGrande <br> 5-Shot |70.80|82.50|81.40|54.7|54.20|55.60|62.00|68.80|
+| OpenBookQA <br> 10-Shot |84.20|88.40|87.20|73.6|79.80|78.60|85.80|86.00|
+| BoolQ <br> 0-Shot |77.60|82.90|86.60| -- |72.20|66.00|76.60|79.10|
+| CommonSenseQA <br> 10-Shot |80.20|80.30|82.60|69.3|72.60|76.20|78.10|79.60|
+| TruthfulQA <br> 10-Shot |65.00|68.70|75.70| -- |53.00|52.10|60.10|67.70|
+| HumanEval <br> 0-Shot |59.10|59.10|55.50|59|28.00|34.10|37.80|62.20|
+| MBPP <br> 3-Shot |53.80|71.40|74.50|60.6|50.80|51.50|60.20|77.80|
+
+## Inference samples
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation-dolly" target="_blank">text-generation-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation-dolly" target="_blank">text-generation-online-endpoint.sh</a>
```

Comment:
Do we want to create separate ones for these models?

Reply:
Noted Sasi. I will document this bug in the loop page. Since we are not taking the release to production, shall we address this in the follow up PR?

Reply:
If we can establish that there are no issues with the latest artifacts, PST folks will be in a good position to incorporate all these changes for the prod release.